docs/variant_annotation.md

```diff
@@ -101,6 +101,18 @@ followed by `_vep_output.vcf`. Note that if this directory already exists, then
 Variant Transforms fails. This is to prevent unintentional overwriting of old
 annotated VCFs.
 
+* [`--shard_variants`](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/options/variant_transform_options.py#L290)
+by default, the input files are sharded into smaller temporary VCF files before
+running VEP annotation. If the input files are small, i.e., each VCF file
+contains less than 50,000 variants, set this flag to false can be more time
+efficient.
+
+* [`--number_of_variants_per_shard`](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/options/variant_transform_options.py#L360)
+the maximum number of variants written to each shard if `shard_variants` is
+true. The default value should work for most cases. You may change this flag to
+a smaller value if you have a dataset with a lot of samples. Notice that it may
+take a longer time to run with a smaller value.
```

Comment:
S/.../Notice that pipeline may take longer to finish for smaller value of this flag.